brainfm/__init__.py

```diff
@@ -221,7 +221,21 @@ def operation(self, **kwargs):
     },
     # getMainStations  # not implemented for now, use get_stations
     # getMainStations_NoPlans  # not implemented for now, use get_stations
-    # getStationsByID  # unknown; not seen
+    "get_stations_by_id": {
+        "name": "getStationsByID",
+        "method": "post",
+        "parameters": [
+            {
+                "name": "pid",
+                "alias": "parent_id",
+                "required": True,
+                "type": int
+            }
+        ],
+        "response": jmespath.compile(
+            "{stations: stations}"),
```

Comment:
What's the structure for each item in this list?  This might be optimistic, but if it mirrors `get_station` then we should apply the same transform (so the example wouldn't have to use the wire format `id` and instead keep us informed `station_id` everywhere)